### PR TITLE
XRT-2858: update opc-ua server test with new mapping model

### PR DESCRIPTION
**Pull request by JonathanWhiteUK**

### CODE REVIEW

File: Servers/opc-ua/federated/modbus-tcp/deployment/state/modbus/schedules.json
Comment (dpazj) [inline Servers/opc-ua/federated/modbus-tcp/deployment/state/modbus/schedules.json:1]: Formatting is a bit off

File: Servers/opc-ua/node-modeller/deployment/config/opc_ua_server_config.json
Comment (dpazj) [inline Servers/opc-ua/node-modeller/deployment/config/opc_ua_server_config.json:1]: Can we rename this folder "Servers/opc-ua/**node-modelling**"

**Review by dpazj — CHANGES_REQUESTED** [review]

File: Servers/opc-ua/node-modeller/deployment/config/opc_ua_server_config.json
Comment (JonathanWhiteUK) [reply Servers/opc-ua/node-modeller/deployment/config/opc_ua_server_config.json:1]: will do now.

presume you want that done on 2.2 too

File: Servers/opc-ua/federated/modbus-tcp/deployment/state/modbus/schedules.json
Comment (JonathanWhiteUK) [reply Servers/opc-ua/federated/modbus-tcp/deployment/state/modbus/schedules.json:1]: Resolved

File: Servers/opc-ua/node-modeller/deployment/config/opc_ua_server_config.json
Comment (JonathanWhiteUK) [reply Servers/opc-ua/node-modeller/deployment/config/opc_ua_server_config.json:1]: done for 3.0

**Review by dpazj — APPROVED** [review]
LGTM

**Pull request merged**